DataFormats/L1TMuon/src/EMTFHit.cc

```diff
@@ -73,18 +73,28 @@ namespace l1t {
   }
 
   CSCCorrelatedLCTDigi EMTFHit::CreateCSCCorrelatedLCTDigi() const {
-    return CSCCorrelatedLCTDigi(1,
-                                valid,
-                                quality,
-                                wire,
-                                strip,
-                                pattern,
-                                (bend == 1) ? 1 : 0,
-                                bx + CSCConstants::LCT_CENTRAL_BX,
-                                0,
-                                0,
-                                sync_err,
-                                csc_ID);
+    CSCCorrelatedLCTDigi lct = CSCCorrelatedLCTDigi(1,
+                                                    valid,
+                                                    quality,
+                                                    wire,
+                                                    strip,
+                                                    pattern,
+                                                    (bend == 1) ? 1 : 0,
+                                                    bx + CSCConstants::LCT_CENTRAL_BX,
+                                                    0,
+                                                    0,
+                                                    sync_err,
+                                                    csc_ID);
```

Comment:
Shouldn't you also (conditionally) update the `version` in `lct` for Run3 (the unmodified default is "legacy")?
Maybe it is already done elsewhere...